### DISCUSSION TITLE
Area Filler

### DISCUSSION THREAD

**Discussion by anarchy1337x** [PrismarineJS/mineflayer#2487] [Q&A]
I've recently had the idea of making multiple bots simultaneously fill in an area, but i have no clue where to start.

I've labeled 4 points on a grid and each bot goes to their corresponding points, how would i find each (x,y,z) coordinate inside the 4 given points?

Thanks for any feedback that i may get!

**Comment by amoraschi** [comment]
Do 3 for loops inside each other
```
for y
  for x
    for z
```

So they loop through all coordinates

**Reply by anarchy1337x** [reply]
Could you give me an example?

(e.g for the coords)
-50, 10, 59
-60, 10, 59
-100, 10, 59
-140, 10, 59

(not exact coords)

the problem i'm facing is filling in the area from them coords.

**Reply by amoraschi** [reply]
https://www.w3schools.com/js/js_loop_for.asp

**Reply by anarchy1337x** [reply]
No i didn't mean that. I meant can you break it down.  (and tell me why you got the answer)

**Comment by anarchy1337x** [comment]
Also i'm not too sure that's what i'm looking for. I'm looking for the area (and to triangulate from 4 points)
